### PR TITLE
Update FAQ.md

### PR DESCRIPTION
**Pull request by sahamidzade**
---

## Code Review Checklist

- [ ] Description accurately reflects what changes are being made.
- [ ] Description explains why the changes are being made (or references an issue containing one).
- [ ] The PR appropriately sized.
- [ ] New code has enough tests. 
- [ ] New code has enough documentation to answer "how do I use it?" and "what does it do?".
- [ ] Existing documentation is up-to-date, if impacted.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,7 +1,129 @@
...
+No iOS support: we currently do not offer an iOS version of the application. 
+
+Unable to set default file destination for Android: a user is not able to change the default directory in Android, so files are all automatically saved to the default Downloads folder. 
+
+Interoperability challenges: While we are striving to ensure interoperability between Destiny and other applications using the Magic Wormhole protocol, this has not fully been tested. As such, we cannot at this stage guarantee full interoperability.  
+
+We will work on addressing these issues in V2 of the application. We welcome external contributions to the code and improvements. In the meantime, please report any other bugs and suggestions to destiny@leastauthority.com. 
+
+
+**Contact**
```
Comment (donpui) [inline FAQ.md:123]: We need to have headline: ## CONTACT
as I user in url in app: https://github.com/LeastAuthority/destiny/blob/main/FAQ.md#contact

File: FAQ.md [hunk 2]
```diff
@@ -1,7 +1,143 @@
...
+
+**At what point can I retract a file that I am sending?** 
+
+As a sender you can first cancel a transfer before sharing the code by pressing the Cancel button. After sharing the code you can still press Cancel if the other party has not yet started downloading the file. If you press Cancel while the transfer is ongoing, it is possible for the receiver to already have a portion of the file.
+
+**Can I use Destiny to send/receive files to/from other Magic Wormhole applications?**  
+
+Yes. Make sure to configure the other Magic Wormhole application to use the same environment settings set in Destiny. These are currently:
+
+	Mailbox URL: wss://mailbox.w.leastauthority.com/v1
```
Comment (donpui) [inline FAQ.md:116]: We have new address:
Mailbox: wss://mailbox.mw.leastauthority.com/v1
Relay: tcp://relay.mw.leastauthority.com:4001

**Review by donpui — CHANGES_REQUESTED** [review]
Please update addresses

File: FAQ.md (hunk 2 above)
Comment (sahamidzade) [reply FAQ.md]: updated

**Pull request merged**